packages/ralph/skills/golem-powers/figma-loop/SKILL.md

```
@@ -0,0 +1,153 @@
+|---------------------|----------|
+| Full iteration loop (start to finish) | [workflows/iterate.md](workflows/iterate.md) |
+| Set up tracking file | [workflows/setup.md](workflows/setup.md) |
+| Just do a single comparison check | [workflows/check.md](workflows/check.md) |
+
+---
+
+## The 3-Check Rule
+
+A component is only "done" when **3 consecutive checks pass with zero changes needed**.
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)